lib/ecstatic.js

```diff
@@ -124,13 +124,26 @@ var ecstatic = module.exports = function (dir, options) {
               url: parsed.pathname + '.' + defaultExt + ((parsed.search)? parsed.search:'')
             }, res, next);
           }
-          else {
-            // Try to serve default ./404.html
+          else if (!handleError) {
+            // If we're not handling errors/fallbacks, bail out now
```

Comment:
...is this a bug? I'll have to look more closely, but I think maybe handleError: false should have us be calling next directly.